### PR TITLE
I/O for AbsLine

### PR DESCRIPTION
**Pull request by profxj**
Small edits for AbsLine I/O

Test modified too

no docs

### CODE REVIEW

File: linetools/spectralline.py [hunk 1]
```diff
@@ -126,8 +127,9 @@ def from_dict(cls, idict, coord=None, warn_only=False, chk_data=True, **kwargs):
                 #pdb.set_trace()
                 sline.analy[key] = ltu.convert_quantity_in_dict(idict['analy'][key])
             elif key == 'spec_file':
-                warnings.warn("You will need to load {:s} into attrib['spec'] yourself".format(
+                warnings.warn("You will need to load {:s} into analy['spec'] yourself".format(
                         idict['analy'][key]))
```
Comment (ntejos) [inline linetools/spectralline.py]: Can you add a commented line explaining that analy['spec'] is expected to be `XSpectrum1D` and `analy['spec_file']` is the filename of the spectrum.

File: linetools/spectralline.py [hunk 2]
```diff
@@ -482,7 +487,11 @@ def fill_data(self, trans, linelist=None, closest=False, verbose=True):
 
         # Data
         newline = llist[trans]
-        self.data.update(newline)
+        try:
```
Comment (ntejos) [inline linetools/spectralline.py]: Please add commented line explaining that newline is expected to be a LineList dict() output.

**Comment by ntejos** [comment]
Looks good; please consider adding these commented lines to the code for future developers.

**Comment by profxj** [comment]
Merging.

**Pull request merged**